numismatic/feeds/base.py

```diff
@@ -141,9 +147,12 @@ def __requester_validator(self, attribute, value):
         elif not isinstance(value, Requester):
             raise ValueError(f'{attribute.name}: {value}')
 
-    def _make_request(self, api_url, params=None, headers=None):
+    def _make_request(self, api_url, params=None, headers=None, raw=False):
         response = self.requester.get(api_url, params=params, headers=headers)
-        data = response.json()
+        if not raw:
```

Comment:
Nit: Can you flip this? Easier to read without not:
```
if raw:
   data = response
else:
   data = response.json()
```

Reply:
I agree, however ... in this case I think the not raw case is more often so I wanted the most common path to come first.